pkg/render/compliance.go

```diff
@@ -374,11 +388,19 @@ func (c *complianceComponent) complianceControllerClusterAdminClusterRoleBinding
 }
 
 func (c *complianceComponent) complianceControllerDeployment() *appsv1.Deployment {
+	var keyPath, certPath string
+	if c.cfg.ControllerKeyPair != nil {
+		// This should never be nil, but we check it anyway just to be safe.
+		keyPath, certPath = c.cfg.ControllerKeyPair.VolumeMountKeyFilePath(), c.cfg.ControllerKeyPair.VolumeMountCertificateFilePath()
+	}
+
 	envVars := []corev1.EnvVar{
 		{Name: "LOG_LEVEL", Value: "info"},
 		{Name: "TIGERA_COMPLIANCE_JOB_NAMESPACE", Value: ComplianceNamespace},
 		{Name: "TIGERA_COMPLIANCE_MAX_FAILED_JOBS_HISTORY", Value: "3"},
 		{Name: "TIGERA_COMPLIANCE_MAX_JOB_RETRIES", Value: "6"},
+		{Name: "LINSEED_CLIENT_CERT", Value: certPath},
```

Comment:
Don;t we also need the CA passed in ?

Reply:
The CA is defaulted in the code, so we don't need to set that explicitly here.

Reply:
That's how it has been done for other components as well